subprojects/diagnostics/src/main/java/org/gradle/api/tasks/diagnostics/AbstractReportTask.java

```diff
@@ -71,6 +72,10 @@ public void generate() {
             @Override
             public void generateReport(Project project) throws IOException {
                 generate(project);
+
+                if (!outputsToConsole()) {
+                    project.getLogger().lifecycle("See the report at: file://{}", TextUtil.normaliseFileSeparators(getOutputFile().getAbsolutePath()));
```

Comment:
This should use `org.gradle.internal.logging.ConsoleRenderer#asClickableFileUrl`